docs/INSTALL.md

```diff
@@ -0,0 +1,138 @@
+## Installation
+
+Ensure KUBECONFIG points to the target Kubernetes cluster
+```
+kubectl apply -f https://raw.githubusercontent.com/confidential-containers/operator/ccv0-demo/deploy/deploy.yaml
+```
+
+## Create Custom Resource (CR)
+```
+kubectl apply  -f https://raw.githubusercontent.com/confidential-containers/operator/ccv0-demo/config/samples/ccruntime.yaml
```

Comment:
This PR uses a [file](https://raw.githubusercontent.com/confidential-containers/operator/ccv0-demo/config/samples/ccruntime.yaml) from `ccv0-demo` branch. But the [current demo doc](https://github.com/confidential-containers/operator/blob/ccv0-demo/demo/README.md#install-confidential-containers-runtime) on the `ccv0-demo` points to the [config](https://raw.githubusercontent.com/confidential-containers/operator/main/config/samples/ccruntime.yaml) from `main` branch.

Is this intentional?

Reply:
This is intentional. I'll remove the `demo` folder in a follow on PR.